snowflake/1.0/modules/ROOT/pages/snowflake-connector-studio.adoc

```diff
@@ -0,0 +1,124 @@
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener source in the app.
+. On the *General* tab, specify connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for Snowflake Connector, follow these steps:
+
+. In *Mule Palette*, select *Snowflake* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Snowflake system. Snowflake Connector supports <mention the types of authorization the connector supports>.
+
+To configure the global element for Snowflake Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click *+* next to the *Connector configuration* field to access the global element configuration fields.
+. In the *General* tab, in *Required Libraries*, click *Configure* and select from the following options to install the required JDBC driver:
+* *Add recommended libraries*
+* *Use local file*
+* *Add Maven dependency*
+. In *Connection*, enter the following required information to configure authentication:
+* *Account name* +
+Account name that was provided when registering for the Snowflake account
+* *Warehouse* +
+Name of the virtual Snowflake warehouse to use. The specified warehouse must be an existing warehouse for which the specified account has privileges.
+* *Database* +
+Snowflake database to connect to. The specified database must be an existing database for which the specified account has privileges.
+* *Schema* +
+Snowflake database schema to use. The specified schema must be an existing schema for which the specified account has privileges.
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```

Comment:
I know this is boilerplate, but...well...you know.... We always see new things each time.... Possible change to avoid modal verbs:
```suggestion
Either reference a configuration file that contains ANT-style property placeholders (recommended) or enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```
OR
```suggestion
Reference a configuration file that contains ANT-style property placeholders. Alternatively, enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```